include/seqan3/io/alignment_file/format_bam.hpp

```diff
@@ -1147,7 +1147,8 @@ inline std::string format_bam::get_tag_dict_str(sam_tag_dictionary const & tag_d
         {
             // always choose the smallest possible representation [cCsSiI]
             bool negative = arg < 0;
-            auto n = __builtin_ctz(detail::next_power_of_two(((negative) ? arg * -1 : arg) + 1) >> 1) / 8;
+            size_t absolute_arg = ((negative) ? arg * -1 : arg);
```

Comment:
Do we, by chance, already include `cmath` along the way?
Then we could just use `std::abs`.
Performance-wise it seems to do not matter: https://quick-bench.com/q/v68HcJ-1VMgtHkwPORt5VeAfcB8
But it would be less code.

Reply:
I'm not against this, but `negative` is used later.

Reply:
Well, we could replace the one negative with `arg < 0`. 
To be honest, the block is not so clear on why things happen (`n * n - 2 * n * negative` ??).

It may make it easier to read (using `std::abs`), but you can do as you see fit.